packages/server/src/utils/livepeer.ts

```diff
@@ -76,7 +76,7 @@ export const createAsset = async (
   }
 };
 
-export const getAsset = async (
+export const getPlayback = async (
```

Comment:
There are now two getAssets. One of them is quite misleading, since it just returns `playbackUrl`

Reply:
Just changed it a bit... if playbackUrl exists the phase is always "ready". So there is no use of phase anymore once you have the playbackUrl

Reply:
I merged getVideoPhaseAction and getPlayback to one function(getAsset)